protocol-units/execution/opt-executor/src/transaction_pipe.rs

```diff
@@ -358,7 +434,7 @@ mod tests {
 	#[tokio::test]
 	async fn test_repeated_pipe_mempool_from_api() -> Result<(), anyhow::Error> {
 		let (tx_sender, mut tx_receiver) = mpsc::channel(16);
-		let (executor, config, _tempdir) = Executor::try_test_default(GENESIS_KEYPAIR.0.clone())?;
+		let (executor, _config, _tempdir) = Executor::try_test_default(GENESIS_KEYPAIR.0.clone())?;
```

Comment:
I noticed these warnings, too. Returning the config has become useless in that test function since it's part of `Executor` state now anyway.